.github/workflows/test-library.yml

```
@@ -195,6 +195,12 @@ jobs:
             && github.event.inputs.release-version
             || steps.scm-version.outputs.dist-version
         }}-py3-none-any.whl')
+    - name: Set container version
+      id: container-version
+      shell: bash
+      run: >-
+        VER=$(echo '${{ needs.pre-setup.outputs.dist-version }}' | tr + .);
+        print('::set-output name=version::$VER')
```

This is copy-pasted from the Python block. Convert everything to Python or switch the syntax here to bash:
```suggestion
        echo "::set-output name=version::$VER"
```